mozilla-release/toolkit/components/extensions/webrequest/WebRequest.jsm

```diff
@@ -723,6 +723,25 @@ HttpObserverManager = {
         this.handleForgetModeNotification(otherWin);
       }
 
+      try {
```

Comment:
@tsl143 Please add our comments tag #CLIQZ-SPECIAL
Also I thought that it might be a good idea to not only add the tag itself but with the browser version at the end as well.
#CLIQZ-SPECIAL-1.34

Reply:
Also could you please tell which part might throw an error here? Why do you need try/catch?

Reply:
same as below comment